nix/haskell.nix

```diff
@@ -91,9 +94,8 @@ haskell-nix.cabalProject' ({ pkgs
       })
       ({ pkgs, ... }: {
         # Use the VRF fork of libsodium
-        packages = lib.genAttrs [ "cardano-crypto-praos" "cardano-crypto-class" ] (_: {
-          components.library.pkgconfig = lib.mkForce [ [ pkgs.libsodium-vrf ] ];
-        });
+        packages.cardano-crypto-class.components.library.pkgconfig = lib.mkForce [[ pkgs.libsodium-vrf pkgs.secp256k1 ]];
```

Comment:
Shouldn't this be fixed once and for all in the relevant packages? There's been a lot of work on pkgconfig stuff in haskell.nix lately, cc @hamishmack